kolibri/plugins/coach/assets/src/views/common/resourceSelection/subPages/SelectionIndex.vue

```diff
@@ -1,6 +1,12 @@
 <template>
 
   <div>
+    <QuizResourceSelectionHeader
+      v-if="target === SelectionTarget.QUIZ && !settings.selectPracticeQuiz"
+      class="mb-24"
+      :settings="settings"
+      @searchClick="onSearchClick"
+    />
     <div
```

Comment:
We will also need to take out these headers to fix it in lessons and for practice quizzes too :open_hands:

https://github.com/nucleogenesis/kolibri/blob/d497f89cec0ebf7776ba4b5d32ebc403885314bd/kolibri/plugins/coach/assets/src/views/common/resourceSelection/subPages/SelectionIndex.vue#L14-L37

And I guess in lessons we should render this string conditionally only if we have bookmarks https://github.com/nucleogenesis/kolibri/blob/d497f89cec0ebf7776ba4b5d32ebc403885314bd/kolibri/plugins/coach/assets/src/views/common/resourceSelection/subPages/SelectionIndex.vue#L19

Reply:
@AlexVelezLl just pushed a fix for these two, thanks a ton Alex!